x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/step_configure_package.test.tsx

```diff
@@ -150,4 +154,17 @@ describe('StepConfigurePackage', () => {
     });
     expect(mockUpdatePackagePolicy.mock.calls[0][0].inputs[0].enabled).toEqual(false);
   });
+
+  it('should render without data streams or vars', async () => {
+    packageInfo.data_streams = [];
+    packagePolicy.inputs[0].streams = [];
```

Comment:
This causes the validation service to return `inputs: null` which is the state that triggered the initial error.

Reply:
Sure I can make that change as it's quite small and doesn't have much impact with the refactors included here. It seems safer to return `{}` than `null`, though - based on how `inputs` is used elsewhere in the code. 

I'll also add an FTR test to make sure a `POST` to the package policy API succeeds for a package with a single input and no data streams.